operator.go

```diff
@@ -30,28 +32,33 @@ func main() {
 		logrus.Panic(err)
 	}
 
-	clientset, err := versioned.NewForConfig(config)
+	extClientset, err := apiextensionsclient.NewForConfig(config)
 	if err != nil {
 		logrus.Panic(err)
 	}
 
+	err = crd.MySQLClusterCreateCRD(extClientset)
+	if err != nil {
+		panic(err)
+	}
+
+	clientset, err := versioned.NewForConfig(config)
+	if err != nil {
+		panic(err)
+	}
+
 	ctx, cancelFunc := context.WithCancel(context.Background())
 	defer cancelFunc()
 
 	clusterController := controller.NewClusterController(clientset)
 	go clusterController.Run(ctx)
 
-	go logging.LogEvents(
```

Comment:
This part is pretty important atm not only for logging, but it prevents clusterController channel from blocking completely after 100 events.